CMakeLists.txt

```diff
@@ -107,7 +107,7 @@ set(BEEROCKS_BRIDGE_IFACE "br-lan")
 if (NOT DEFINED BEEROCKS_BH_WIRE_IFACE)
     set(BEEROCKS_BH_WIRE_IFACE "eth1")
 endif()
-set(BEEROCKS_NOTIFY 0)
+set(BEEROCKS_NOTIFY 1)
```

Comment:
not sure if this configuration is needed , i think it was used in dummy mode before we had hostapd .
If you decide keeping at least change it to BEEROCKS_IFACE_NOTIFY

Reply:
This is was used to notify the beerocks SL - this is no longer needed, so why did you add it? It is not clear.

Reply:
The platform manager needs to know the status. It uses it.

Reply:
Removed completly